js/sum.js

```diff
@@ -0,0 +1,46 @@
+let addBtn = document.getElementById('add-btn');
```

Comment:
- This implementation is lacking "functional programming" common approaches. 
Its easier to call it procedural.